MyApplication/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.justinwells.myapplication">
+
+    <application
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+        <activity android:name=".MainActivity">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+    </application>
```

Comment:
You need to add `<service android:name=".MyMediaService"/>` inside `<application></application>`! Without this, your service will never be started, and the buttons do nothing!

See https://github.com/ga-adi-macaron/Course-Materials/tree/master/lessons/android-technologies-and-services/services-lesson#demo-create-a-service-15-mins